src/main/java/com/kireyko/movieland/controller/MovieController.java

```diff
@@ -0,0 +1,115 @@
+package com.kireyko.movieland.controller;
+
+import com.kireyko.movieland.entity.Genre;
+import com.kireyko.movieland.entity.Movie;
+import com.kireyko.movieland.service.MovieService;
+import com.kireyko.movieland.util.JsonJacksonConverter;
+import com.kireyko.movieland.util.JsonManualConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/v1")
+public class MovieController {
+    private int movieid = 0;
+    private int genreId =110;
+    private final Logger log = LoggerFactory.getLogger(getClass());
+
+    private static final String COMMA_SEPARATOR = ",";
+    private static final String COLON_SEPARATOR = ":";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
+    @Autowired
+    private MovieService movieService;
+
+    @Autowired
+    private JsonManualConverter jsonConverter;
+
+    @Autowired
+    private JsonJacksonConverter jsonJacksonConverter;
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
+    public String welcome(ModelMap model) {
+        model.addAttribute("movieid", ++movieid);
+        model.addAttribute("genreId", genreId);
+
+        log.debug("[movie_] movieid : {}", movieid);
+        return "v1";
+    }
+
+    @RequestMapping(value = "/movie", method = RequestMethod.GET, produces = "application/json;charset=utf-8" )
```

Comment:
1) produces, method can be moved into class @RequestMapping
2) for produces better to use constants